tools/install

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+ROOT=$(dirname $0)/..
+(cd $ROOT && npm install)
+(cd $ROOT && npm install)
```

Comment:
why does this happen twice?

Reply:
Must have been a copy and paste mistake while adding ROOT, nice catch, LOL.